server/nodejs/resources/js/index.js

```diff
@@ -2,7 +2,7 @@ document.addEventListener('DOMContentLoaded', () => {
     const surveyData = [
         {
             survey_id: 1,
-            survey_title: 'Sample Survey 1',
+            survey_title: 'Sample Survey 1  ',
```

Comment:
Why does this have a space?

Reply:
my bad, maybe i left it for testing purposes on ui